sdk/src/androidTest/java/ly/count/android/sdk/ModuleAPMTests.java

```diff
@@ -174,6 +175,36 @@ public void customTrace() {
         Assert.assertTrue((stop.getValue() - start.getValue()) < 100);
     }
 
+    /**
+     * Test that custom trace key is truncated to the correct length
+     * Validate custom metrics are merged and truncated to the correct length
+     * Validate that the custom trace is sent to the server with correct values
+     */
+    @Test
+    public void customTrace_truncate() {
```

Comment:
you should also validate the cancel call to make sure it doesn't break that usage